lib/publiccloud/provider.pm

```diff
@@ -452,14 +452,9 @@ sub terraform_apply {
     my $ips;
     my $resource_id;
     if (get_var('PUBLIC_CLOUD_SLES4SAP')) {
-        foreach my $vm_type ('cluster_nodes', 'drbd', 'netweaver') {
-            if (is_azure && $vm_type eq 'cluster_nodes') {
-                push @{$vms}, @{$output->{$vm_type . '_name'}->{value}[0]};
-                push @{$ips}, @{$output->{$vm_type . '_public_ip'}->{value}[0]};
-            } else {
-                push @{$vms}, @{$output->{$vm_type . '_name'}->{value}};
-                push @{$ips}, @{$output->{$vm_type . '_public_ip'}->{value}};
-            }
+        foreach my $vm_type ('hana_public', 'drbd', 'netweaver') {
+            push @{$vms}, @{$output->{$vm_type . '_name'}->{value}};
+            push @{$ips}, @{$output->{$vm_type . '_public_ip'}->{value}};
```

Comment:
Nice that we can have all CSP in the same format again. We may need to change this later for the new deployment if the outputs are different though.

Reply:
> Nice that we can have all CSP in the same format again. We may need to change this later for the new deployment if the outputs are different though.

I think we can adapt changes into new deployment since the tf code is almost the same.

Reply:
taking care that we already have something : https://github.com/SUSE/qe-sap-deployment/commit/e1326ca695da39432283335f5a23b6456f4696cb

Reply:
Got a reply from @michaelgrifalconi that this can be ignored for now as they're still debugging TruffleHog.